content/events/2020-05-28-virtualiotandedgeday2020.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Eclipse Virtual IoT and Edge Native Day 2020"
+date: "2020-05-28T14:00:00-00:00"
```

Comment:
This is for the publish date for this page of content.

This needs to be set to a now or a previous date for it to appear on the site!

Reply:
The event is showing in the previous since this server is designed to include unpublished content for testing. https://deploy-preview-25--eclipsefdn-sparkplug.netlify.app/

Reply:
https://github.com/EclipseFdn/sparkplug.eclipse.org/blob/master/netlify.toml#L18

Reply:
@chrisguindon Looks like I goofed, then. Can you edit the file so that it will be displayed right away? Thanks.

Reply:
@chrisguindon Looks like I goofed. Thanks for catching this. I corrected this PR and the two others.

Reply:
Thanks @fdesbiens !